modules/administration/pages/openscap.adoc

```diff
@@ -37,23 +45,22 @@ For more information about the {suse} Security Team, see  https://www.suse.com/s
 == Prepare Clients for an SCAP Scan
 
 Before you begin, you need to prepare your client systems for SCAP scanning.
-Ensure clients have the ``openscap-content`` and ``openscap-utils`` packages installed before you begin.
-
 
+For scanning {sles} clients, install the ``openscap-content`` and ``openscap-utils`` packages before you begin.
 Use this command to determine the location of the appropriate SCAP files.
-Take a note of these paths for performing the scan:
-+
+Take a note of the file paths for performing the scan:
+
 ----
 rpm -ql openscap-content
```

Comment:
Wondering whether you can do those package listings with the Web UI?

Reply:
Maybe with "run remote command" :-(

From what i can see, in "minion details -> Audit -> Schedule, the field "Path to XCCDF document" is a free input text field, so it's not suggesting a valid value. You have to run such a command to know what to enter there. Of course a drop-down with the valid values would make user"s life easier.

Reply:
Something for the UI team :wink: